### PR TITLE
Updates SDK and Mockito versions

### PR DESCRIPTION
**Pull request by naft-a**
Dependency issues with Dart SDK 2.1.0-dev.0.0

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -5,10 +5,10 @@ author: Ian Luca de Godoy Serpa <ian.naph@gmail.com>
 homepage: https://github.com/ianldgs/material_search
 
 environment:
-  sdk: '>=1.8.0 <2.0.0'
+  sdk: '>=1.8.0 <=2.1.0'
```
Comment (ianldgs) [inline pubspec.yaml]: Maybe removing this limitation would be better, don't you think? What happens when dart releases 2.2?

Comment (ianldgs) [reply pubspec.yaml]: Mind following these instructions?
https://github.com/ianldgs/material_search/issues/30#issuecomment-418119069
If you can't, at my lunch time today I fix it...

Comment (naft-a) [reply pubspec.yaml]: No rush, I will try to change it later today. Thanks!

Comment (ianldgs) [reply pubspec.yaml]: Great! Thank you very much for the PR.

**Pull request merged**